### DISCUSSION TITLE
python3 netbox/manage.py migrate ModuleNotFoundError: No module named 'django' when upgrading NetBox

### DISCUSSION THREAD

**Discussion by infraengineer-sre** [netbox-community/netbox#5491] [Q&A]
### Environment
* Python version: Python 3.6.8
* NetBox version: v2.7.4

### Steps to Reproduce
1. i got an error 
[root@NetBox netbox-2.7.12]# ./upgrade.sh
Removing old virtual environment...
Creating a new virtual environment at /opt/netbox-2.7.12/venv...
Error: Command '['/opt/netbox-2.7.12/venv/bin/python3', '-Im', 'ensurepip', '--upgrade', '--default-pip']' returned non-zero exit status 1.
--------------------------------------------------------------------
ERROR: Failed to create the virtual environment. Check that you have
the required system packages installed and the following path is
writable: /opt/netbox-2.7.12/venv
--------------------------------------------------------------------

2. i created virtualenv with "--without-pip, activate/deactivate and  curl https://bootstrap.pypa.io/get-pip.py" workaround
3. i tried to run commands one by one

### Expected Behavior
`(venv) [root@NetBox netbox-2.7.12]# python3 netbox/manage.py migrate`

### Observed Behavior
```
Traceback (most recent call last):
  File "netbox/manage.py", line 8, in <module>
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'
```

**Comment by ryanmerolle** [comment]
Did you ever sort this out?
